include/ozo/impl/connection.h

```diff
@@ -161,6 +161,24 @@ inline auto get_native_handle(const Connection& conn) noexcept {
     return unwrap_connection(conn).native_handle();
 }
 
+template <typename Connection>
+inline const auto& get_error_context(const Connection& conn) {
+    static_assert(ozo::Connection<Connection>, "T must be a Connection");
```

Comment:
"T must be a Connection" -> "conn should model Connection"?

Reply:
**Done**